src/modules/Packages.rb

```diff
@@ -29,6 +29,13 @@ class PackagesClass < Module
     # Minimum set of packages required for installation with remote X11 server
     REMOTE_X11_BASE_TAGS = [ "xorg-x11-server", "xorg-x11-fonts", "icewm" ]
 
+    # Some products are already be "included" in other products. So they MUST
+    # not be installed anymore because they are conflicting to the other one.
+    PRODUCT_CONFLICTS = {
+      # SLES_SAP includes SLES. So SLES is not needed anymore
+      "SLES" => [ "SLES_SAP" ]
```

Comment:
Actually as we dropped the 2nd stage we should update the code from `return true if Stage.cont` to `return true unless Stage.initial`.